src/Microsoft.VisualStudio.Services.Agent/Util/VarUtil.cs

```diff
@@ -70,10 +70,12 @@ public static string OSArchitecture
         /// Example: env.var -> ENV_VAR
         /// </summary>
         /// <param name="value"></param>
+        /// <param name="preserveCase"></param>
         /// <returns></returns>
-        public static string ConvertToEnvVariableFormat(string value)
+        public static string ConvertToEnvVariableFormat(string value, bool preserveCase = false)
```

Comment:
It's easier to review the callsites are updated with preserveCase: false, rather than using a default value for the argument

Reply:
Updated all call sites and made parameter explicit